client/src/components/grants/Show.tsx

```diff
@@ -27,20 +27,23 @@ function Project() {
   }, shallowEqual);
 
   useEffect(() => {
-    if (props.ipfsInitialized && params.id) {
+    // called twice
```

Comment:
Maybe we can consolidate this and https://github.com/gitcoinco/grants-hub/pull/72/files#diff-98c909ece664b6cfe734fbadcfe5749ebdc02e5d20b98e02526bb09d98370900R80 into a component and use it while IPFS is loading? It could remove some of the conditions in the components. Or even a toast? wdyt?